katago_server/trainings/models/network.py

```diff
@@ -41,11 +67,16 @@ class Meta:
         verbose_name = _("Network")
         verbose_name_plural = _("Networks")
 
+    # QUESTION (lightvector): Do any changes need to be made such that this can be a different identifier supplied by the training process?
```

Comment:
If it doesn't match an uuid, I should only convert it as Charfield

Reply:
I'm not sure what this means, but I'll let you handle this. So if the training machine is going to upload a network, presumably the API will also have the training machine at runtime tell the server through the API call what the model's name is, which could be used as the key, and additionally at that time the server would also call `parse_katago_training_model_name` on it to extract out possible the other details, if present.

Reply:
Nevermind, I just went ahead and tried this myself. Still not sure where `parse_katago_training_model_name` should be called, but maybe even easier is if the training machine also just provides these values separately?